src/movies.js

```diff
@@ -1,28 +1,76 @@
+//let fs = require("fs");
+const movies = require("./data");
+//let Data = fs.readFileSync("./data.js", "utf-8");
+
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors() {
+  let Directors = []
+  movies.forEach(element => {
+    Directors.push(element.Directors)
+  });
+  return Directors;
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies() {}
+function howManyMovies() {
+  let counter = 0;
+  movies.forEach(element => {
+    if (element.Directors == 'Steven Spielberg')
+    {
+      counter++;
+    }
+  });
+  return counter;
+}
```

Comment:
- This iteration should be solved using `filter() `method ;) 
- Task asks to count all 'Drama' movies by Spielberg. So you need to be checking both director and genre